source/content/tmp.md

```diff
@@ -16,18 +17,31 @@ export env=dev
 </Alert>
 
 ## Default Temporary Path
+
 Pantheon configures an appropriate temporary path for [WordPress](https://github.com/pantheon-systems/WordPress/blob/4.9.6/wp-config.php#L83-L86) and [Drupal 8](https://github.com/pantheon-systems/drops-8/blob/8.5.3/sites/default/settings.pantheon.php#L146-L154). Drupal 7 sites can achieve the same configuration by adding the following to `settings.php`:
 
-```php
+```php:title=settings.php
 /**
  * Drupal 7
  * Define appropriate location for tmp directory
  */
 if (isset($_ENV['PANTHEON_ENVIRONMENT'])) {
-  $conf['file_temporary_path'] = $_SERVER['HOME'] .'/tmp';
+
+  // Pressflow will override whatever we set here unless we override Pressflow first.
+
+  $pressflow = json_decode($_SERVER['PRESSFLOW_SETTINGS'], TRUE);
+
+  $pressflow['conf']['file_temporary_path'] = $pressflow['conf']['file_directory_temp'] = '/tmp';
+
+  $_SERVER['PRESSFLOW_SETTINGS'] = json_encode($pressflow);
+
 }
 ```
+
+Change `sites/default/files/private/tmp` to your desired temporary file path.
```

Comment:
I didn't notice the recommendation here to put the private files directory in Valhalla; note that the code snippet above is inconsistent with this advice, recommending `/tmp` instead.